octopus.py

```diff
@@ -79,15 +79,16 @@ def get_account_number(token: str) -> str:
 
 
 GET_HH_BODY = """
-query halfHourlyReadings($accountNumber: String!, $fromDatetime: DateTime, $toDatetime: DateTime) {
+query halfHourlyReadings($accountNumber: String!, $fromDatetime: DateTime, $toDatetime: DateTime, $productCode: String!) {
```

Comment:
You'd also need to update the place where this is used I think - down on line 117

Reply:
@FraserTooth  did you mean around here?
<img width="467" alt="Screenshot 2023-01-26 at 9 49 13" src="https://user-images.githubusercontent.com/45124890/214729402-4a64fe69-95b0-4150-9cce-190843a42413.png">

Reply:
https://github.com/octoenergy/oejp-api-example/pull/2#discussion_r1087332516